deploy/examples/images.txt

```diff
@@ -1,9 +1,9 @@
- k8s.gcr.io/sig-storage/csi-attacher:v3.4.0
- k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
- k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
- k8s.gcr.io/sig-storage/csi-resizer:v1.4.0
- k8s.gcr.io/sig-storage/csi-snapshotter:v5.0.1
- k8s.gcr.io/sig-storage/nfsplugin:v3.1.0
+ registry.k8s.io/sig-storage/csi-attacher:v3.4.0
+ registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.0
+ registry.k8s.io/sig-storage/csi-provisioner:v3.1.0
+ registry.k8s.io/sig-storage/csi-resizer:v1.4.0
+ registry.k8s.io/sig-storage/csi-snapshotter:v5.0.1
+ registry.k8s.io/sig-storage/nfsplugin:v3.1.0
```

Comment:
The images.txt seems to be complaining that the images should be in a different order now.
```
diff --git a/deploy/examples/images.txt b/deploy/examples/images.txt
index 1cdefe697..a869d9b41 100644
--- a/deploy/examples/images.txt
+++ b/deploy/examples/images.txt
@@ -1,11 +1,11 @@
+ quay.io/ceph/ceph:v16.2.7
+ quay.io/cephcsi/cephcsi:v3.6.0
+ quay.io/csiaddons/k8s-sidecar:v0.2.1
+ quay.io/csiaddons/volumereplication-operator:v0.3.0
  registry.k8s.io/sig-storage/csi-attacher:v3.4.0
  registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.0
  registry.k8s.io/sig-storage/csi-provisioner:v3.1.0
  registry.k8s.io/sig-storage/csi-resizer:v1.4.0
  registry.k8s.io/sig-storage/csi-snapshotter:v5.0.1
  registry.k8s.io/sig-storage/nfsplugin:v3.1.0
- quay.io/ceph/ceph:v16.2.7
- quay.io/cephcsi/cephcsi:v3.6.0
- quay.io/csiaddons/k8s-sidecar:v0.2.1
- quay.io/csiaddons/volumereplication-operator:v0.3.0
  rook/ceph:master
```